From 9f97da53b791c6590a51208bfeb2265efaf7fdae Mon Sep 17 00:00:00 2001
From: zhang sanshan <b51434@freescale.com>
Date: Tue, 17 Nov 2015 10:54:36 +0800
Subject: [PATCH] MA-7226 [Android_6QP_SD]G-sensor: The intial orientation of
 6QP_SD is different with 6Q_SD, 90 degree difference. 100%

The position of MMA8451Q on 6qp board is not the same with 6q.
Correct the init position value in 6qp sabresd dts.

Signed-off-by: zhang sanshan <b51434@freescale.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index a77b375..4b0fc03 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -51,6 +51,11 @@
 	};
 };
 
+&i2c1 {
+	mma8451@1c {
+		position = <1>;
+	};
+};
 &mxcfb1 {
 	prefetch;
 };
-- 
1.8.0

